styx-common/src/main/java/com/spotify/styx/util/CachedSupplier.java

```diff
@@ -18,10 +18,9 @@
  * -/-/-
  */
 
-package com.spotify.styx;
+package com.spotify.styx.util;
```

Comment:
This was just a product of my ignorance :)

I think it does exactly the same as the guava momoized supplier. Feel free to remove it.